tensorboard/webapp/util/ui_selectors_test.ts

```diff
@@ -265,6 +265,63 @@ describe('ui_selectors test', () => {
           ])
         );
       });
+
+      it('does not violently throw when an experiment metadata DNE', () => {
```

Comment:
What is DNE?

Reply:
Does not exist. It was terminology I used when doing math. If it is too obscure, I will change it. Let me know what you prefer.

Reply:
Yes it seems a little obscure.

Reply:
Done.